packages/code-block/partials/code-tabs/index.js

```diff
@@ -23,14 +25,31 @@ function CodeTabs({ children, heading, className, tabs, theme = 'dark' }) {
   // the expected CodeBlock, CodeBlockConfig, or pre
   const childTypes = validChildren.map((tabChild) => {
     let type
-    // For JSX primitives, the type of captured by the type property
-    if (typeof tabChild.type == 'string') type = tabChild.type
-    // For function components in JSX, ie CodeBlock and CodeBlockConfig,
-    // tabChild.type is a function whose name we need
-    if (typeof tabChild.type == 'function') type = tabChild.type.name
-    // In MDX contexts, the component type is captured in mdxType
-    if (typeof tabChild.props.mdxType === 'string')
+    // For JSX primitives, the type is captured by the type property
+    if (typeof tabChild.type == 'string' || typeof tabChild.type == 'number') {
+      type = tabChild.type
+      // For function components, accept CodeBlock or CodeBlockConfig.
+    } else if (typeof tabChild.type === 'function') {
```

Comment:
This builds on the approach Bryce suggested to maintain the existing behaviour of flagging all invalid types in a an error message. Due to the minification concern, there's a bit of repetition here to make sure the intended names make it into the error message; my thinking is that's a decent tradeoff for having clearer, easier-to-debug error messages.

Have also made a small style change to use braces for the `if` blocks 👍